examples/compressed_inference/fp8_compressed_inference.py

```diff
@@ -25,9 +23,10 @@
     "def fibonacci(n):",
 ]
 
-# set run_compressed=True to enable running in compressed mode
-compressed_model = SparseAutoModelForCausalLM.from_pretrained(
-    MODEL_STUB, torch_dtype="auto", device_map="cuda:0", run_compressed=True
+compressed_model = AutoModelForCausalLM.from_pretrained(
+    MODEL_STUB,
+    torch_dtype="auto",
+    device_map="cuda:0",
 )
```

Comment:
Is there a way we can load the model in uncompressed mode?

Reply:
Load is dependent on HFQuantizer, need to edit that to load in uncompressed